conformance-suites/2.0.0/conformance/extensions/webgl-compressed-texture-etc.html

```diff
@@ -138,10 +139,18 @@
     shouldBe("formats.length", isPositive ? "10" : "0");
 
     debug("");
-    shouldThrow("gl.compressedTexImage2D(gl.TEXTURE_2D, 0, COMPRESSED_R11_EAC, 4, 4, 0, null)");
-    shouldThrow("gl.compressedTexSubImage2D(gl.TEXTURE_2D, 0, 0, 0, 0, 0, COMPRESSED_R11_EAC, null)");
-    shouldThrow("gl.compressedTexImage3D(gl.TEXTURE_2D_ARRAY, 0, COMPRESSED_R11_EAC, 4, 4, 4, 0, null)");
-    shouldThrow("gl.compressedTexSubImage3D(gl.TEXTURE_2D_ARRAY, 0, 0, 0, 0, 0, 0, 0, COMPRESSED_R11_EAC, null)");
+    if (contextVersion >= 2) {
+      // `null` coerces into `0` for the PBO entrypoint, yielding INVALID_OP due to no PBO bound.
```

Comment:
Yunchao,

If you update your chromium source, you will see the new function signatures being added to the IDL, compressed* with offset instead of srcData.  So null will be 0 and will match these newly added signatures.